.setup/submitty_websocket_server.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Submitty WebSocket Server
+Documentation=http://submitty.org
+
+[Service]
+User=submitty_php
+Group=submitty_php
+Restart=on-failure
```

Comment:
I dont really know the difference but in the other submitty service files they use `always` instead of `on-failure` here. Is that something that should be switched?

Reply:
yeah, probably. changed to `always`